elytron/src/test/resources/org/wildfly/extension/elytron/realms-test.xml

```diff
@@ -55,6 +55,11 @@
         <token-realm name="OAuth2Realm" principal-claim="sub">
             <oauth2-introspection client-id="a" client-secret="b" introspection-url="https://localhost/token/introspect" client-ssl-context="ClientCaSslContext" host-name-verification-policy="ANY" />
         </token-realm>
+
+        <jaas-realm name="myJaasRealm"  entry="${jaas.realm.login.context.entry}" >
```

Comment:
I suppose a default value can be used, e.g. `${jaas.realm.login.context.entry:Entry1}`.

All the attributes allowing expressions should be tested with an expression set (somewhere). (`path`, `module`, and `callback-handler`?)

Reply:
@OndrejKotek I added path to be resolved from expression. Both use default value as you suggested. Btw module, callback handler and relative-to do not allow expressions.